Backend/Backend/Services/EpisodeService.cs

```diff
@@ -81,12 +128,19 @@ public async Task<bool> DeleteEpisode(Episode episode, DeleteEpisodeRequest dele
 
                         }
 
-                        bool? editted = await  _fileService.EditFile(episode.AudioFile!, editEpisodeRequest.AudioFile);
-                        if(editted == false)
+                        bool? editted =  _fileService.EditFile(episode.AudioFile!, editEpisodeRequest.AudioFile);
+                        if(editted == true)
                         {
-                            throw new Exception("Failed to Edit the file");
+                            episode.AudioFile!.Name = editEpisodeRequest.AudioFile.FileName;
```

Comment:
I know this is not the reponsibility of this PR, but you should probably rewrite this code so that it isnt indented 4 levels right off the bat. you can use guard clauses for that.

Reply:
heres an example of guard clauses 
![image](https://github.com/awaazo/awaazo/assets/78050750/cd21cf0b-3389-4c1d-a7bd-a5ccbc120287)